llvm/lib/CodeGen/MLRegAlloc/README.md

```diff
@@ -0,0 +1,111 @@
+# RL4ReAl
+
+`RL4ReAl` is a retargetable Reinforcement Learning (RL) approach for solving the REgister ALlocation (REAL) problem on diverse architectures.
+
+
+
+This repo contains the source code and relevant information described in the [paper](https://dl.acm.org/doi/abs/10.1145/3578360.3580273) ([arXiv](https://arxiv.org/abs/2204.02013)).
+Please see [here](https://compilers.cse.iith.ac.in/publications/rl4real/) for more details.
+
+> RL4ReAl: Reinforcement Learning for Register Allocation, S. VenkataKeerthy, Siddharth Jain, Anilava Kundu, Rohit Aggarwal, Albert Cohen and Ramakrishna Upadrasta
+
+## Environment Setup
+
+
+- Setup the environment using the `.yml` using the following commands
+
+    ```bash
+    conda env create -f rl4real_env.yml
+    ```
+
+## Pre-Trained Model Runners for Inference
+
+- Currently three different model runners are supported my the `MLCompilerBridge` 
```

Comment:
1. Write something like, "There are three modes for inference supported via MLCompilerBridge"
2. Remove the pre-trained from title